source_control/subversion.py

```diff
@@ -103,7 +110,7 @@ def __init__(
         self.password = password
         self.svn_path = svn_path
 
-    def _exec(self, args):
+    def _exec(self, args, check_rc=True):
         bits = [
```

Comment:
Document the changes to this function - having a parameter change what the function returns could be confusing. Something along the lines of:

    Execute a subversion command, and return output. If check_rc is False, returns the return code instead of the output.